config-check/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>com.okta.commons</groupId>
         <artifactId>okta-commons-root</artifactId>
-        <version>1.2.8-SNAPSHOT</version>
+        <version>1.2.9-SNAPSHOT</version>
```

Comment:
Should we release as 1.3.0 since we have a new feature?

Reply:
I'd thought the same but the semver (https://github.com/okta/okta-commons-java/blob/master/pom.xml#L179-L180) doesn't consider this necessary.

Reply:
sure will do.